src/registers/control.rs

```diff
@@ -1,6 +1,8 @@
 //! Functions to read and write control registers.
 
 pub use super::model_specific::{Efer, EferFlags};
+#[cfg(docsrs)]
+use crate::{registers::rflags::RFlags, structures::paging::PageTableFlags};
```

Comment:
What is this about?

Reply:
This is needed for the doc comments to properly reference these other structures (like `PageTableFlags::GLOBAL`). Without them, the docs don't properly link to the other sections.

Reply:
Oh ok. But there imports are then also needed for local doc builds, e.g. on `cargo doc --open`. So we should probably use `cfg(doc)` instead. I opened https://github.com/rust-osdev/x86_64/pull/287 for this.